beets/ui/commands.py

```diff
@@ -489,6 +494,13 @@ def choose_candidate(candidates, singleton, rec, cur_artist=None,
         assert cur_artist is not None
         assert cur_album is not None
 
+    # Build helper variables for extra choices.
+    # TODO: move "abort" choice always to the end?
+    # TODO: add plugin name info on choices ("(mb) Print tracks")?
+    extra_opts = tuple([c.long for c in extra_choices])
+    extra_actions = {c.long.strip(lowercase + whitespace)[0].lower(): c.id
```

Comment:
Got it; this makes sense. Either way (implicit or explicit) seems fine—if we go implicit, we'll just have to make it clear in the docs that the capitalized letter is how you control your action letter.